dradis/dradis-ce
# add activity tracking to comments

### Spec
Add activity tracking to comments, as we have for other resources

**Proposed solution**
Try to use different approach: use callbacks in the model that trigger a bg task to create the corresponding activity.

### How to test
Use the rails console to add a comment to an issue. Maybe like this:    
Comment.create(commentable: Issue.first, user: User.first, content: "hello")
The main page, assert we can see the activity.
Edit + delete the issue, check those activities too
